en-US/Design.xml

```diff
@@ -637,12 +637,12 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
           </listitem>
           <listitem>
             <para>
-              As much as possible, leave the system-wide default as <varname>become: false</varname> or <varname>become: no</varname> and if a single task needs privileges, set <varname>become: true</varname> or <varname>become: yes</varname> on that task.
+              As much as possible, configure the default setting for <varname>become</varname> to <code>false</code>, and if a single task needs privileges, set <varname>become: true</varname> on that task.
```

Comment:
```suggestion
              As much as possible, configure the default setting for <varname>become</varname> to <code>false</code>. If a single task needs privileges, then set <varname>become: true</varname> on that task.
```

Reply:
Thanks for clarifying the context. I suggest breaking this line into 2 sentences for clarity.

Reply:
Y'all can sort that out.  :smiley:   I was just trying to make the one practical change that was needed, without worrying about re-wordsmithing the content.

Reply:
@bonnevil Sure, I understand. I'll tag @julian-cable so that we can resolve this. Thanks for getting the ball rolling on this.